package.json

```diff
@@ -65,6 +69,7 @@
     ]
   },
   "dependencies": {
-    "cross-fetch": "^3.1.5"
+    "cross-fetch": "^3.1.5",
+    "dotenv": "^16.3.1"
```

Comment:
I believe this should be under devDep. I see that you only use this for the test file?

Reply:
Yes that's right, I'll move this to dev.

Reply:
@yen-tt Are you referring just to dotenv or also crossfetch?

Reply:
dotenv only. I see that cross-fetch is used in the HttpRequester class so it's in the right dep type